playbooks/azure/leapp-rhui-azure.spec

```
@@ -0,0 +1,64 @@
+# Update name of package
+%global provider    azure
+Name:           leapp-rhui-%{provider}
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Leapp in-place upgrade cloud specific package
+
+# Provider variable is name of subdirectory that is cloud specific, please
+# refer to Leapp documentation/code for what subdir you should use.
+# Examples: aws, azure
+%global leappfilespath  %{_datadir}/leapp-repository/repositories/system_upgrade/el7toel8/files/rhui
+
+License:        LGPLv2+
+URL:            http://redhat.com
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}
+
+BuildArch:      noarch
+
+Requires:       leapp leapp-repository
+
+# Update description
+%description
+Leapp in-place upgrade on %{provider}
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+
+%install
+rm -rf $RPM_BUILD_ROOT
+
+
+mkdir -p %{buildroot}/%{leappfilespath}/%{provider}
```

@mreznik remove the slash before the macro everywhere, so the result is not `/<buildroot>//usr/...`, but `/<buildroot>/usr/...`

e.g.
```suggestion
mkdir -p %{buildroot}%{leappfilespath}/%{provider}
```

---

like you did not tell before. Got it now...